packages/formation/sass/modules/_va-pagination.scss

```diff
@@ -12,54 +11,122 @@
   display: flex;
   justify-content: center;
 
+  button {
+    -moz-osx-font-smoothing: auto;
+    -webkit-font-smoothing: auto;
+    background-color: transparent;
+    border: 0;
+    color: $color-link-default;
+    display: inline-flex;
+    font-size: 0.9em;
+    font-weight: 400;
+    justify-content: center;
+    line-height: 2;
+    padding: 0;
+    transition: all 0.3s ease-in-out;
+    transition-property: color, background-color;
```

Comment:
Are all of these styling changes based on a design? I'm not sure if they're representing uswds styling, something in app code, etc.